GG/GG/Font.h

```diff
@@ -147,6 +151,13 @@ class GG_API Font
             pair.second. */
         Substring(const std::string& str_, const IterPair& pair);
 
+        /** Attach this Substring to \p str_.
+
+            This changes the iterators from pointing into the previous
```

Comment:
It won't actually invalidate the iterators, though, as SubString just stores a pointer to a strong. What would invalidate would be if the pointed-to-string was manipulated in such a way that its iterators are invalidated. Code that uses this might cause this, but not this function in of itself... (I think...)